textile/features.textile

```diff
@@ -1655,6 +1656,17 @@ h4. DevicePushDetails
 * @(PCP3)@ recipient - a map of string key/value pairs containing details of the push transport and address
 * @(PCP4)@ state - the state of the push registration, one of  @Active@, @Failing@, @Failed@
 
+h3(#introspection). Client library introspection
+
+h4. ClientLibraryInfo
+* @(CLI1)@ Provides information about the client library.
+* @(CLI2)@ @agents@ static property:
+** @(CLI2a)@ Returns a @Dict<String, String?>@ that lists the default key-value entries that the library uses to populate the @Agent@ library identifier, as described by "@RSC7d@":#RSC7d.
+* @(CLI3)@ @agentIdentifier@ static method:
+** @(CLI3a)@ Returns the @Agent@ library identifier described by "@RSC7d@":#RSC7d.
+** @(CLI3b)@ If the library offers the @ClientOptions#agents@ property described by "@RSC7d6@":#RSC7d6, then the @agentIdentifier@ method must accept an @additionalAgents@ parameter of type @Dict<String, String?>?@, whose value is to be interpreted with the same semantics as the @ClientOptions#agents@ property. The @agentIdentifier@ method will inject these agents into the @Agent@ library identifier that it returns.
+** @(CLI3c)@ An API commentary must be provided for the @additionalAgents@ parameter, if present. This commentary must make it clear that only Ably-authored SDKs should pass a non-null value for this parameter.
```

Comment:
I was doing it to be consistent with [RSC7d6b](https://sdk.ably.com/builds/ably/specification/main/features/#RSC7d6b), which states that the concept of specifying additional agents is exclusively for Ably-authored SDKs.

Do you think it would be better for me to change CLI3c to say that this whole method should be exclusively for Ably-authored SDKs?

Reply:
OK, I may have misread this a bit the first time around, but yeah I think it would be better if the method itself was labelled as Ably-only 👍

Reply:
I've added that as CR3d.